index.js

```diff
@@ -85,8 +93,13 @@ function link(done) {
   var links = [
     path.join('.bin', 'eslint'),
     'eslint',
+    'babel-preset-latest',
```

Comment:
I'd prefer using `babel-preset-env` which I think is now recommended over `latest`.